packages/blade/src/components/Accordion/AccordionButton.native.tsx

```diff
@@ -1,18 +1,28 @@
 import type { ReactElement } from 'react';
-import { StyledAccordionButton } from './StyledAccordionButton';
+import { StyledAccordionButton } from './StyledAccordionButton.native';
```

Comment:
```suggestion
import { StyledAccordionButton } from './StyledAccordionButton';
```

Reply:
It wasn't picking the typing correctly (i.e. `Pressable`) 😅 

So I changed it to `.native` explicitly because we're inside a native module